test/python/quantum_info/operators/symplectic/test_sparse_pauli_op.py

```diff
@@ -161,6 +161,21 @@ def test_from_list(self):
         np.testing.assert_array_equal(spp_op.coeffs, coeffs)
         self.assertEqual(spp_op.paulis, PauliList(labels))
 
+    def test_from_index_list(self):
+        """Test from_list method specifying the Paulis via indices."""
+        labels = ["XXZ", "IXI", "YIZ", "III"]
+        paulis = ["XXZ", "X", "YZ", ""]
+        indices = [[0, 1, 2], [1], [0, 2], []]
+        coeffs = [3.0, 5.5, -1j, 23.3333]
+        spp_op = SparsePauliOp.from_list(list(zip(paulis, indices, coeffs)), num_qubits=3)
```

Comment:
type hint notwithstanding, the code is clearly intended to work with non-list iterables, so the `list(zip(...))` could just be `zip(...)` (and the type-hint updated).